src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -125,4 +125,24 @@ public String getNewQuestion(Entity game_entity, DatastoreService datastore) {
         return (update_game_entity.getProperty("quizQuestion")).toString();
     }
 
+    public Boolean giveUserPoints(Boolean userQuizStatus, Entity currentUser, DatastoreService datastore) {
```

Comment:
I think boolean should be lowercase here

Reply:
If I change it to lower case "b" I get an incompatible type error

Reply:
https://programming.guide/java/boolean-vs-boolean.html

We should use lowercase b here. Likely, the reason you're getting incompatibility errors is because if you only change it to lowercase b here but not everywhere else, the compiler will get angry.